app/models/ocelot/stanzas/ValueStanza.scala

```diff
@@ -40,6 +40,7 @@ object Value {
 }
 
 case class ValueStanza(values: List[Value], override val next: Seq[String], stack: Boolean) extends Stanza {
+  override val visual: Boolean = false
```

Comment:
As discussed, can we define a NonVisualStanza extending Stanza and setting visual = false.  Then any non-visual stanza extends this new stanza so they don't have to override visual attribute

Reply:
will add